14LessonFourteen/in-class-project/css/style.css

```diff
@@ -0,0 +1,172 @@
+body {
+  color: #333;
+  text-align: center;
+  font-family: "Roboto Mono";
+}
+
+.navbar-brand img {
+  position: absolute;
+  top: 0;
+}
+
+.navbar-fixed-top {
+  top: -1px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: "Work Sans", sans-serif;
+}
+
+.hero {
+  background-image: url("http://bone.themanoid.com/images/office/03.jpg");
+  background-size: cover;
+  background-position: 50%;
+  height: 100vh;
+  padding: 120px 0;
+  position:relative;
+  min-height: 500px;
+}
+
+.target {
+  height: 50px;
+  position:absolute;
+  bottom: 0;
+
+}
+
+h1 {
+  font-size: 5em;
+  margin-top: 15vh;
+}
+
+h2 {
+  font-size: 2.2em;
+  font-family: "Roboto Mono";
+  font-weight: 500;
+  line-height: 1.1;
+  margin: 75px 25px;
+}
+
+h3 {
+  margin-top: 0;
+  margin-bottom: 25px;
+  font-size: 1.7em;
+  font-family: "Work Sans";
+}
+
+.grid {
+  height: 600px;
+  position: relative;
+}
+
+.grid-item {
+  position: absolute;
+  width: 33.3333%;
+}
+
+.grid-item::before {
+  content: '';
+  display: block;
+  padding-top: 75.001%;
+}
+
+.grid .tall::before {
+  padding-top: 150%;
+}
+
+.grid-item:nth-child(1), .grid-item:nth-child(4) {
+  left: 0%;
+}
+
+.grid-item:nth-child(2), .grid-item:nth-child(5) {
+  left: 33.254%;
+}
+
+.grid-item:nth-child(3) {
+  left: 66.5873%;
+}
+
+.grid-item:nth-child(n+4) {
+  top: 293px;
+}
```

Comment:
next time try relying on [bootstrap's column offset](http://getbootstrap.com/css/#grid-offsetting) to fine-tune your positioning. It will save you lots of time in the long run!